### PR TITLE
auto transcribe new messages, add requirements.txt

### PR DESCRIPTION
**Pull request by strazto**

### CODE REVIEW

File: src/cogs/transcribe.py [hunk 1]
```diff
@@ -61,9 +54,108 @@ async def transcribe(self, ctx, *, translate_to=None): # takes an optional argum
...
+        return embed
+
+
+def msg_has_voice_note(msg: typing.Optional[discord.Message]) -> bool:
+    if not msg: return False
+    if not msg.attachments or not msg.flags.value >> 13: return False
+    return True
+
+async def transcribe_msg(msg: typing.Optional[discord.Message]) -> typing.Optional[typing.Union[typing.Any,list,tuple]]:
+        if not msg_has_voice_note(msg): return None
```
Comment (dromzeh) [inline src/cogs/transcribe.py]: change to `if not msg or msg_has_voice_note(msg): return None`?

**Review by dromzeh — CHANGES_REQUESTED** [review]

File: src/cogs/transcribe.py (hunk 1 above)
Comment (strazto) [reply src/cogs/transcribe.py]: Can do, if you like, but `msg_has_voice_note(msg)` already returns falsy if `msg` is `None`

**Comment by strazto** [comment]
Btw, most significant change besides UX change from just autotranscribing, I changed the attributions in the embed -
I remove the Google / deepl attributions, and link back to this repo, as well as our profiles


Screenshot_20230712_104045_Discord

Anyone who cares about the Google / deepl stuff can find out via the readme, which also boosts your profile hahaha.

File: src/cogs/transcribe.py (hunk 1 above)
Comment (dromzeh) [reply src/cogs/transcribe.py]: > Can do, if you like, but `msg_has_voice_note(msg)` already returns falsy if `msg` is `None`

yeah, just suggested this as we should probably check for message before accessing the attributes?(https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportOptionalMemberAccess)

Comment (strazto) [reply src/cogs/transcribe.py]: https://github.com/strazto/vmt/blob/c41be4996bb2d31f45657ea96bbc21d5d86421f8/src/cogs/transcribe.py#L151 this line checks message is defined before accessing attributes too :) 

I'd say a boolean `or` short circuit statement is no less astonishing than a function that returns `False` if the input it undefined

Comment (strazto) [reply src/cogs/transcribe.py]: I don't have pyright set up in my env, is it giving an warning?

Comment (dromzeh) [reply src/cogs/transcribe.py]: > I don't have pyright set up in my env, is it giving an warning?

yes - `if not msg...` removes that

Comment (strazto) [reply src/cogs/transcribe.py]: done :)

**Review by dromzeh — APPROVED** [review]

**Review by dromzeh — APPROVED** [review]
lgtm :)

**Pull request merged**